spec/factories/series.rb

```diff
@@ -5,14 +5,17 @@
     title { "John" }
     code  { "BL-123" }
     image_link { "" }
+    editors { "NBC" }
+    description { "World" }
+    founder { "Washington" }
 
-    trait :with_image do
-      after :create do |series|
-        series.image.attach(io:
-          File.open(Rails.root.join("spec/fixtures/charles.jpg")),
-          filename: "charles.jpg",
-          content_type: "image/jpeg")
-      end
-    end
+    # trait :with_image do
```

Comment:
Why is this one commented out?

Reply:
This is all still a work in progress and not meant to be a finished product. As we may need it in the future, I left it commented. The series work has not begun yet other than setting up the model, and all this code will be iterated over many times as we develop in consultation with the press. Series may not even have an image in the end.  I will remove this and any other not yet implemented commented code.